backend/prisma/migrations/20250630172210_updating/migration.sql

```diff
@@ -0,0 +1,24 @@
+/*
+  Warnings:
+
+  - The values [CANCELED] on the enum `AppointmentStatus` will be removed. If these variants are still used in the database, this will fail.
+  - You are about to drop the column `role` on the `User` table. All the data in the column will be lost.
+
+*/
+-- AlterEnum
+BEGIN;
+CREATE TYPE "AppointmentStatus_new" AS ENUM ('PENDING', 'CONFIRMED', 'CANCELLED', 'RESCHEDULED', 'COMPLETED');
+ALTER TABLE "Appointment" ALTER COLUMN "status" TYPE "AppointmentStatus_new" USING ("status"::text::"AppointmentStatus_new");
+ALTER TYPE "AppointmentStatus" RENAME TO "AppointmentStatus_old";
+ALTER TYPE "AppointmentStatus_new" RENAME TO "AppointmentStatus";
+DROP TYPE "AppointmentStatus_old";
+COMMIT;
+
+-- AlterTable
+ALTER TABLE "Appointment" ALTER COLUMN "status" SET DEFAULT 'PENDING';
+
+-- AlterTable
+ALTER TABLE "User" DROP COLUMN "role";
```

Comment:
I am a bit confused here, if you remove the 'role', then how do you do the role base access control later?